src/components/calcite-input-message/calcite-input-message.scss

```diff
@@ -64,3 +64,30 @@
 :host([status="idle"]) .calcite-input-message-icon {
   color: var(--calcite-ui-brand);
 }
+
+// Validation Text
+:host([status][active]) {
+  @apply text-color-1 
+    font-medium;
+}
+
+:host([status][active][scale="s"]) {
+  @apply text--3;
```

Comment:
I think the font sizes need the `h` suffix based on what I see in Figma

Reply:
Yes ^